fs.go

```diff
@@ -45,17 +47,18 @@ func IsDir(name string) (bool, error) {
 	return true, nil
 }
 
-func IsNonEmptyDir(name string) (bool, error) {
-	isDir, err := IsDir(name)
-	if !isDir || err != nil {
-		return isDir, err
-	}
-
+// IsEmptyDirOrNotExist is true if name is a directory and is not empty or
+// doesn't exist.
+// Returns an error when name is a file or on other fs/io errors.
```

Comment:
nit: I believe an extra newline is needed to get these to appear as separate paragraphs in godoc